Calculator/Calculator/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Christy Lee on 2023/01/24.
+//
+
+import Foundation
+
+struct CalculatorItemQueue<T> {
+    var data = [T] ()
```

Comment:
- 이 data 프로퍼티는 외부에서 접근이 가능한 프로퍼티인가요 ?
- 보통 배열의 초기화는 이렇게 공백을 주지 않고 작성을 한답니다
    ```suggestion
        var data = [T]()
    ```

Reply:
현재는 외부에서 접근이 가능하지만 이 부분은 외부에서 접근하지 않아도 될 것 같습니다. 다음 스텝을 진행하며 `private (set)` 을 적용하겠습니다! 공백의 경우 제거했습니다!